src/components/AppHeader.jsx

```diff
@@ -45,10 +50,31 @@ const AppHeader = ({ classes }) => (
       <Button color="inherit" component={Link} to="/profile">
         My Fundraising
       </Button>
+      {deleteDatabase && (
+        <Button
+          color="inherit"
+          onClick={async () => {
+            const accessToken = await getAccessToken();
+            fetch(`${API}/DeleteTheFreakingDatabaseNOW`, {
+              method: 'get',
+              headers: {
+                'content-type': 'application/json',
+                accept: 'application/json',
+                authorization: `Bearer ${accessToken}`,
+              },
+            });
+          }}
+        >
+          Seek And Destrooooooy!
+        </Button>
+      )}
```

Comment:
why do we need this @Ioannis-Karasavvaidis ?

Reply:
Fast way to clear the database on the fly? You need to set an env var to display this.

Reply:
This seems a little overkill to me.  My preference would be to run 'psql -u postgres DROP DATABASE step-tracker` if required

Reply:
after DROPing the database need to create it again :/

Reply:
‘psql -u postgres DROP DATABASE step-tracker;CREATE DATABASE step-tracker;’

Reply:
I don’t think it should go in. It would be a disaster if the data gets deleted during the challenge (only thing it offers is a minor convenience imho)

Reply:
I don’t think it should go in. It would be a disaster if the data gets deleted during the challenge (only thing it offers is a minor convenience imho)